src/Storage/TSDatabaseView.m

```diff
@@ -20,12 +20,10 @@
 
 NSString *TSThreadDatabaseViewExtensionName  = @"TSThreadDatabaseViewExtensionName";
 NSString *TSMessageDatabaseViewExtensionName = @"TSMessageDatabaseViewExtensionName";
-NSString *TSThreadIncomingMessageDatabaseViewExtensionName = @"TSThreadOutgoingMessageDatabaseViewExtensionName";
```

Comment:
Just to be clear, this will also eventually affect the "share profile" behavior.  But this naming issue is resolved.